### ISSUE TITLE
Support for GLIBC 2.31

### ISSUE THREAD

**Issue by miikanissi** [miguelmota/cointop#101]
Trying to run latest Cointop binary on Debian Bullseye is not working due to GLIBC 2.32 dependency. Is there a chance to have Cointop support GLIBC version 2.31?
![image](https://user-images.githubusercontent.com/48931463/113334609-92764280-932c-11eb-9e99-fb6733c8c0f9.png)

**Comment by mayur-chavhan** [comment]
same issue from my end.

![image](https://user-images.githubusercontent.com/15307733/115065769-34d91d00-9f0c-11eb-9283-0bda5aefe1c9.png)

Any fix ?

**Comment by miikanissi** [comment]
The issue seems to be with complining the binary. If the binary is compiled with glibc_2.32 it won't be backwards compatible with versions below glibc_2.32. I found an article describing this issue in general: https://www.lordaro.co.uk/posts/2018-08-26-compiling-glibc.html . 
@miguelmota I suggest you try and compile the next binary release using an older version of glibc as currently Ubuntu nor Debian support glibc_2.32.

**Comment by cocoflan** [comment]
Same problem here, on debian X64 an ARM 64 and ARM 32 systems.

How to fix?

Only the virtual systems are working like Docker image or snap.

**Comment by lyricnz** [comment]
@miikanissi how did you install cointop? which version?
I just installed Debian Bullseye and then cointop via
`curl -o- https://raw.githubusercontent.com/cointop-sh/cointop/master/install.sh | bash`
and got the same issue.

I downloaded the source, and built without any problem.

**Comment by miikanissi** [comment]
> @miikanissi how did you install cointop? which version? I just installed Debian Bullseye and then cointop via `curl -o- https://raw.githubusercontent.com/cointop-sh/cointop/master/install.sh | bash` and got the same issue.
> 
> I downloaded the source, and built without any problem.

Yes the issue was/is with the binary release as it has been compiled with a later glibc version compared to Debian Bullsey glibc version. If you build it from source there are no issues.

**Comment by Foul** [comment]
i'm on KDE Neon and impossible to run Cointop too : 

./cointop: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./cointop)

must compil to work.
